app/uk/gov/hmrc/apidocumentation/views/apiIndex.scala.html

```diff
@@ -16,34 +16,19 @@
 
 @import uk.gov.hmrc.apidocumentation.config._
 @import uk.gov.hmrc.apidocumentation.models._
+@import uk.gov.hmrc.apidocumentation.models.APICategory._
+
 @import include._
 
 @(pageAttributes: PageAttributes,
-        exampleApis: Seq[APIDefinition],
-        otherApis: Seq[APIDefinition],
-        testSupportApis: Seq[APIDefinition])(implicit request: play.api.mvc.Request[Any], applicationConfig: ApplicationConfig, messages: Messages)
+        apisByCategory: Map[APICategory, Seq[APIDefinition]])(implicit request: play.api.mvc.Request[Any], applicationConfig: ApplicationConfig, messages: Messages)
 
 @apiMain(pageAttributes) {
 
-        <h1 class="heading-xlarge">API documentation</h1>
-
-    <p>These are the RESTful APIs we offer on the Developer Hub.</p>
-    <p>Separately we also offer <a href="https://www.gov.uk/topic/dealing-with-hmrc/software-development">XML APIs</a>.</p>
-
-    <h2 id="example-api" class="heading-large">Example API</h2>
-
-    @apiList(exampleApis, "example")
-
-    <br/>
-
-    <h2 id="apis" class="heading-large">APIs</h2>
+    <h1 class="heading-xlarge">API documentation</h1>
 
-    @apiList(otherApis, "other")
+    <p>View documentation for our current APIs, roadmaps of future APIs, and services guides on how APIs work together.</p>
```

Comment:
Not sure this makes sense given there are no service guides or roadmaps

Reply:
Good point, I'll see if I can get some interim copy while we wait for service guides and roadmaps